Scripts/Tracking/Follow/Modifier/ModifyRigidbodyVelocity.cs

```diff
@@ -68,38 +68,37 @@ public override void UpdatePosition(Transform source, Transform target)
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to modify.</param>
         /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
The docs are different for this method